src/Pages/FeedDetail/Content.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+
+class Content extends React.Component {
+  render() {
+    const { img, text } = this.props;
+    console.log(img);
+    return (
+      <ContentWrap>
+        <img alt="content" src={img} />
+        <ContentsText> {text} </ContentsText>
+      </ContentWrap>
+    );
+  }
+}
+
+const ContentWrap = styled.div`
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  img {
+    display: flex;
+    width: 100%;
+  }
+`;
+
+const ContentsText = styled.div`
+  margin: 30px 0 30px 0;
+  display: flex;
+  width: 100%;
+  white-space: pre-line;
+`;
```

Comment:
```suggestion
const ContentWrap = styled.div`
  display: flex;
  flex-direction: column;
  width: 100%;
  margin: 0;
    
  img {
    display: flex;
    width: 100%;
  }
`;

const ContentsText = styled.div`
  display: flex;
  width: 100%;
  margin: 30px 0 30px 0;
  white-space: pre-line;
`;
```

이렇게 변경해주시고, ContentWrap에 margin: 0, width: 100% 값이 꼭 필요한 스타일인가요?
ContentsText에도 width: 100% 꼭 필요한 값인지 확인해주세요!